build/media-suite_compile.sh

```diff
@@ -1121,7 +1123,7 @@ if [[ $jpegxl = y ]] && do_vcs "https://gitlab.com/wg1/jpeg-xl.git"; then
     fi
     CXXFLAGS+=" ${extra_cxxflags[*]}" \
         do_cmake global -D{BUILD_TESTING,JPEGXL_{ENABLE_OPENEXR,ENABLE_SKCMS}}=OFF \
-        -DJPEGXL_{FORCE_SYSTEM_BROTLI,STATIC,ENABLE_BENCHMARK}=ON
+        -DJPEGXL_{FORCE_SYSTEM_HWY,FORCE_SYSTEM_BROTLI,STATIC,ENABLE_BENCHMARK}=ON
```

Comment:
What does this do?

Reply:
Google highway was originally inside the jpegxl codec on a third party. Now it is separate library on github.
https://github.com/google/highway

Reply:
And where exactly is highway being installed/compiled?
I see it still as a git submodule in jpeg-xl's repo.

Remove this change.

Reply:
with JPEGXL 0.2.0
```
# Highway
set(HWY_SYSTEM_GTEST ON CACHE INTERNAL "")
if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/highway/CMakeLists.txt" AND
    NOT JPEGXL_FORCE_SYSTEM_HWY)
  add_subdirectory(highway)
  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/highway/LICENSE"
                 ${PROJECT_BINARY_DIR}/LICENSE.highway COPYONLY)
else()
  pkg_check_modules(HWY libhwy)
  if (NOT HWY_FOUND)
    message(FATAL_ERROR
        "Highway library (hwy) not found. Install libhwy-dev or download it "
        "to third_party/highway from https://github.com/google/highway . "
        "Highway is required to build JPEG XL. You can run "
        "${PROJECT_SOURCE_DIR}/deps.sh to download this dependency.")
  endif()
  add_library(hwy INTERFACE IMPORTED GLOBAL)
  if(${CMAKE_VERSION} VERSION_LESS "3.13.5")
    set_property(TARGET hwy PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${HWY_INCLUDE_DIR})
    target_link_libraries(hwy INTERFACE ${HWY_LDFLAGS})
    set_property(TARGET hwy PROPERTY INTERFACE_COMPILE_OPTIONS ${HWY_CFLAGS_OTHER})
  else()
    target_include_directories(hwy INTERFACE ${HWY_INCLUDE_DIRS})
    target_link_libraries(hwy INTERFACE ${HWY_LINK_LIBRARIES})
    target_link_options(hwy INTERFACE ${HWY_LDFLAGS_OTHER})
    target_compile_options(hwy INTERFACE ${HWY_CFLAGS_OTHER})
  endif()
  if(JPEGXL_DEP_LICENSE_DIR)
    configure_file("${JPEGXL_DEP_LICENSE_DIR}/libhwy-dev/copyright"
                   ${PROJECT_BINARY_DIR}/LICENSE.highway COPYONLY)
  endif()  # JPEGXL_DEP_LICENSE_DIR
endif()
```

Reply:
I fail to see how that refutes what I said.
We don't need highway for anything else, and it's not available as a MINGW package, so there's no point in using the system's highway.